Source/WebCore/rendering/RenderLayer.h

```diff
@@ -1339,11 +1339,8 @@ class RenderLayer : public CanMakeWeakPtr<RenderLayer>, public CanMakeCheckedPtr
     WeakPtr<RenderLayer> m_enclosingPaginationLayer;
 
 #if ENABLE(LAYER_BASED_SVG_ENGINE)
-    // Pointer to the enclosing RenderSVGResourceContainer, if present.
-    WeakPtr<RenderSVGResourceContainer> m_enclosingSVGResourceContainer;
-
-    // Pointer to the enclosing RenderSVGHiddenContainer, if present.
-    WeakPtr<RenderSVGHiddenContainer> m_enclosingSVGHiddenContainer;
+    // Pointer to the enclosing RenderSVGHiddenContainer or RenderSVGResourceContainer, if present.
+    WeakPtr<RenderSVGHiddenContainer> m_enclosingSVGHiddenOrResourceContainer;
```

Comment:
Should not it be just `m_enclosingSVGContainer`?

Reply:
That would confuse me, I would expect a RenderSVGContainer as type with your suggested name.

The other comments should be addressed in a follow-up PR, good observations.